### PR TITLE
add the openresty version check

### PR DESCRIPTION
**Pull request by soulsoul**
#207

### CODE REVIEW

**Comment by moonming** [comment]
@soulsoul thanks.

File: bin/apisix [hunk 1]
```diff
@@ -248,6 +248,22 @@ local function read_yaml_conf()
     return yaml.parse(ymal_conf)
 end
 
+local function checkOpenrestyVersion(need_version)
+	local shell="need_ver="..need_version ..";"
```
Comment (membphis) [inline bin/apisix]: style: 
1. bad indentation
2. when we connect a new string, a space is required between the objects: should be `"need_ver=" .. need_version  .. ";"`

File: bin/apisix [hunk 2]
```diff
@@ -339,6 +355,12 @@ function _M.start(...)
     init(...)
     init_etcd(...)
 
+	local need_version="1.15.8";
+	if not checkOpenrestyVersion(need_version) then
+		print("the openresty version must >=", need_version, "\n")
```
Comment (membphis) [inline bin/apisix]: should use `io.stderr:write`, please take a look at here:
https://github.com/iresty/apisix/blob/master/bin/apisix#L293

File: bin/apisix [hunk 3]
```diff
@@ -355,7 +377,7 @@ function _M.reload()
     if os.execute((test_cmd)) ~= 0 then
         return
     end
-
+
```
Comment (membphis) [inline bin/apisix]: if that is an empty line, we'd better use a blank line.

**Pull request merged**